api/src/controllers/translations.ts

```diff
@@ -0,0 +1,95 @@
+import express from 'express';
+import { ForbiddenException } from '../exceptions/index.js';
+import { respond } from '../middleware/respond.js';
+import { validateBatch } from '../middleware/validate-batch.js';
+import { TranslationsService } from '../services/index.js';
+import asyncHandler from '../utils/async-handler.js';
+import type { PrimaryKey } from '@directus/types';
+
+const router = express.Router();
+
+const readHandler = asyncHandler(async (req, res, next) => {
+	const service = new TranslationsService({
+		accountability: req.accountability,
+		schema: req.schema,
+	});
+
+	const result = await service.readByQuery(req.sanitizedQuery);
+
+	res.locals['payload'] = {
+		data: result,
+	};
+
+	return next();
+});
+
+router.search('/', validateBatch('read'), readHandler, respond);
```

Comment:
Do we need this?

Reply:
I kind of thought that we've removed that method from all controllers. But that's not the case, so we might as well keep it here 👍